### PR TITLE
Gave AI a comms console

### PR DESCRIPTION
**Pull request by ScarKy0**
## About the PR

Needs https://github.com/space-wizards/space-station-14/pull/31823 to be merged first

Allows AI to use a comms console as an ability.

## Why / Balance

It's just sorta a thing AI should be able to do.

## Technical details

AI gets access to a comms console ability, which makes a "Station AI Announcement"

## Media

image

## Requirements

- [x] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- add: Station AI now has a comms console ability.
- tweak: Station AI abilities now have a default order.

### CODE REVIEW

**Comment by nymhole** [comment]
just make all PRs depend on eachother so they all get merged, super based. please give the ai the ability to open multiple camera monitors.

**Comment by ScarKy0** [comment]
> just make all PRs depend on eachother so they all get merged, super based. please give the ai the ability to open multiple camera monitors.

I did do that apprently i did smth wrong
Also idk how to do that, for now AI cant even use the camera monitor because it breaks the overlay

**Comment by superjj18** [comment]
Honestly ai should have most of the important consoles like atmos alert, criminal, power, camera, etc.  let the player uses the action menu to decide which they care about having quick access to.

File: Resources/Prototypes/Actions/station_ai.yml [hunk 1]
```diff
@@ -13,24 +14,37 @@
...
+      - type: ShowCriminalRecordIcons
+      #TODO: Add those in when theres no desync issue with ActionComponentChangeEvent
+      #- type: ShowHealthBars
+       # damageContainers:
+        #- Biological
+        #- Inorganic
+        #- Silicon
+      #- type: ShowHealthIcons
+       # damageContainers:
+        #- Biological
```
Comment (slarticodefast) [inline Resources/Prototypes/Actions/station_ai.yml:39]: These are not mentioned in the PR description at all. Make that a separate PR. and don't include commented out code for features that haven't been decided on yet.

**Review by slarticodefast — CHANGES_REQUESTED** [review]

**Comment by slarticodefast** [comment]
Also the changes to Wireless are not part of this PR, but were in another one. Please atomize your PRs better so we don't accidentally merge features that aren't supposed to be part of the PR. Also adjust your PR description.

**Comment by ScarKy0** [comment]
> Also the changes to Wireless are not part of this PR, but were in another one. Please atomize your PRs better so we don't accidentally merge features that aren't supposed to be part of the PR. And adjust your PR description.

Yeah I forgot to update the description since I moved all the Wireless changes to my Fixes PR, my bad

**Review by slarticodefast — APPROVED** [review]

**Pull request merged**